core/src/main/java/feign/RequestTemplate.java

```diff
@@ -670,7 +670,10 @@ void replaceQueryValues(Map<String, ?> unencoded, Map<String, Boolean> alreadyEn
       }
       Collection<String> values = new ArrayList<String>();
       for (String value : entry.getValue()) {
-        if (value.indexOf('{') == 0 && value.indexOf('}') == value.length() - 1) {
+        // Allow a pure param to be represented, e.g. /path/resource?wsdl
+        if (null == value) {
+          values.add(null);
```

Comment:
It's more appropriate to simply skip the value if it's `null` instead of adding a `null` value.

```java
if (value == null) {
   continue;
}
```